packages/ui/cypress/e2e/app/agents/search/index.spec.js

```diff
@@ -0,0 +1,77 @@
+describe('search', () => {
```

Comment:
let's move this file into the read-folder and run these tests through agents/read/index.spec.js (can remove the extra `resetDevelopmentSqlDb`)

also I'd like you to combine the three searches here into one `it()`. you can use template string to write a multi-line it-statement. the reason is that it is a bit faster to run it that way.